docs/hopsworksai/azure/aks_acr_integration.md

```diff
@@ -0,0 +1,229 @@
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-private-dns.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-private-dns.png" alt="Virtual network peering">
+    </a>
+    <figcaption>Private DNS link setup</figcaption>
+  </figure>
+</p>
+
+In the left plane there is a tab called Virtual network links, click on the tab. Next press add.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-vnet-link.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-vnet-link.png" alt="">
+    </a>
+    <figcaption>Private DNS link configuration</figcaption>
+  </figure>
+</p>
+
+Choose a name for the private link and select the virtual network you will use for the hopsworks cluster, then press OK.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-vnet-link-config.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-vnet-link-config.png" alt="">
+    </a>
+    <figcaption>Private DNS link configuration</figcaption>
+  </figure>
+</p>
+
+The setup is now finalized and you can create the hopsworks cluster.
+
+#### Step 1 option b Subnet in AKS network
+
+With this setup the hopsworks cluster will reside in the same virtual network as the AKS cluster. The difference is that a new subnet in the virtual network will be used for the hopsworks cluster.
+
+To setup the subnet, first go to the virtual network that was created by AKS, in our example this was hopsworksstagevnet154. Next go to the subnets tab.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-subnet.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-subnet.png" alt="">
+    </a>
+    <figcaption>AKS subnet setup</figcaption>
+  </figure>
+</p>
+
+Press + Subnet. Choose a name for the subnet, for example "hopsworks" and an IP range that does not overlap with the kubernetes network. Then save.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-subnet-config.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-subnet-config.png" alt="">
+    </a>
+    <figcaption>AKS subnet setup</figcaption>
+  </figure>
+</p>
+
+### Step 2 Create hopsworks cluster
+
+This step assumes you create a cluster in hopsworks.ai The AKS configuration can be set under the Managed containers tab. Select the Use Azure AKS and Azure ACR as enabled. Two new fields with the name of the container registry you created and the AKS cluster name will pop up. In the previous example we created an ACR with name hopsaksregistry and an AKS cluster with name hopsaks-cluster. Hence, the configuration should look similar to the picture below
```

Comment:
assumes you are creating your Hopsworks cluster using hopsworks.ai
or
you have created your Hopsworks cluster using hopsworks.ai